draft-ietf-quic-transport.md

```diff
@@ -968,6 +968,24 @@ signal before advertising additional credit, since doing so will mean that the
 peer will be blocked for at least an entire round trip, and potentially for
 longer if the peer chooses to not send STREAMS_BLOCKED frames.
 
+## Flow Control Performance
+
+Unlike TCP, QUIC decouples flow control from congestion control. This can
```

Comment:
TCP doesn't couple CC and FC either? The receive window is independent of cwnd. (There are some autotuning implementations that couple them, but the spec doesn't.)

Reply:
> TCP doesn't couple CC and FC either? The receive window is independent of cwnd. (There are some autotuning implementations that couple them, but the spec doesn't.)

Exactly right, TCP does not copule them. I know RFC 813.
I wonder if there is another RFC, where the TCP FC is described.